mmv1/templates/terraform/examples/uptime_check_config_cloud_run.tf.erb

```diff
@@ -0,0 +1,33 @@
+resource "google_monitoring_uptime_check_config" "<%= ctx[:primary_resource_id] %>" {
+  display_name = "<%= ctx[:vars]["display_name"] %>"
+  timeout      = "60s"
+  user_labels  = {
+    example-key = "example-value"
+  }
+
+  http_check {
+    path = "some-path"
+    port = "443"
+  }
+
+  monitored_resource {
+    type = "cloud_run_revision"
+    labels = {
+      location     = google_cloud_run_v2_service.<%= ctx[:vars]["cloud_run_primary_id"] %>.location
+      project_id   = google_cloud_run_v2_service.<%= ctx[:vars]["cloud_run_primary_id"] %>.project
+      service_name = google_cloud_run_v2_service.<%= ctx[:vars]["cloud_run_primary_id"] %>.name
+    }
+  }
+}
+
+resource "google_cloud_run_v2_service" "<%= ctx[:cloud_run_primary_id] %>" {
```

Comment:
I would hardcode this instead of using `cloud_run_primary_id`. It may also fix the error

Reply:
Thanks, I don't have any permissions to see the error.
I've updated by hard cording it in this commit https://github.com/GoogleCloudPlatform/magic-modules/pull/10820/commits/f7001ea0ae9de6b10e287da9bb0951a8d8c4690a

Reply:
Sorry, could you actually keep the `cloud_run_service_name: 'cloudrun-service'` line you had before. It ensures that the test resources get cleaned up.

Reply:
Thank you, I have updated it 🙏  https://github.com/GoogleCloudPlatform/magic-modules/pull/10820/commits/dfb33d5c9ac6a9c3a993717f6b4cf2ea4452926c

Reply:
The test is failing, I'll fix it 🙏

Reply:
I updated it, please let me know if I need additional updates!
Thank you!